tests/__init__.py

```diff
@@ -25,17 +25,16 @@
 from redash import limiter, redis_connection
 from redash.app import create_app
 from redash.models import db
-from redash.utils import json_dumps, json_loads
+from redash.utils import json_dumps
 from tests.factories import Factory, user_factory
 
-
 logging.disable(logging.INFO)
 logging.getLogger("metrics").setLevel(logging.ERROR)
 
 
 def authenticate_request(c, user):
     with c.session_transaction() as sess:
-        sess["user_id"] = user.get_id()
+        sess["_user_id"] = user.get_id()
```

Comment:
> Prefix authenticated user_id, remember, and remember_seconds in Flask Session
  with underscores to prevent accidental usage in application code. #470

https://github.com/maxcountryman/flask-login/blob/0.5.0/CHANGES